tests/pyright/test_scalars.py

```diff
@@ -38,35 +57,57 @@ def test_pyright():
 
     # NOTE: This is also guaranteeing that those scalars could be used to annotate
     # the attributes. Pyright 1.1.224+ doesn't allow non-types to be used there
+    __import__("pprint").pprint(results)
```

Comment:
Why are you using `__import__` here?

Reply:
loool, forgot to remove. This is just a macro I have to print things for debugging using `pprint`. In this case, I wanted to get the real output to replace here